core/src/main/scala/com/rackspace/com/papi/components/checker/util/XPathExpressionPool.scala

```diff
@@ -104,9 +104,10 @@ private class XPathExpressionFactory(private val expression : String, private va
 }
 
 
-private class XPath2ExpressionFactory(private val expression : String, private val nc : NamespaceContext) extends XPathExpressionFactory(expression, nc) {
+private class XPathNExpressionFactory(private val expression : String, private val nc : NamespaceContext, private val version : Int) extends XPathExpressionFactory(expression, nc) {
   override def makeObject = {
     val xpath = (new net.sf.saxon.xpath.XPathFactoryImpl()).newXPath()
```

Comment:
The complexity is that XPathNExpressionFactory supports versions 2.0 to 3.1 is backed by Saxon.  XPath 1.0 is backed by the JRE implementation.  Interestingly Saxon doesn't support XPath 1.0, which is why we have two factories.  2.0 is not enteirly backwards compatible with 1.0 in some extream edge cases, so I'm just being safe there.